generator/resources/src/index.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react'
```

Comment:
Since you added this file in here you won't need the replace text steps in the generator